public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -149,25 +150,34 @@ export function ExploreToolbar({ exploreId, topOfViewRef, onChangeTime }: Props)
             <ToolbarButton
               variant="canvas"
               key="split"
-              tooltip="Split the pane"
+              tooltip={t('explore.toolbar.split.tooltip', 'Split the pane')}
               onClick={onOpenSplitView}
               icon="columns"
               disabled={isLive}
             >
-              Split
+              <Trans i18nKey="explore.toolbar.split.title"> Split </Trans>
```

Comment:
Same as the split tooltip. Maybe we could call it `explore.toolbar.split-title` or `explore.toolbar.split-label`

Reply:
Sure and thanks for your feedback. addressed it here.
```suggestion
              <Trans i18nKey="explore.toolbar.split-title"> Split </Trans>
```